media-libs/amdgpu-pro-vulkan/amdgpu-pro-vulkan-21.30.1290604.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit unpacker multilib-minimal
+
+MY_PV_MAJOR=`ver_cut 1`
+MY_PV_MINOR=`ver_cut 2`
+MY_PV_REV=`ver_cut 3`
+MY_PV="${MY_PV_MAJOR}.${MY_PV_MINOR}-${MY_PV_REV}"
+
+DESCRIPTION="AMD's closed source vulkan driver, from Radeon Software for Linux"
+HOMEPAGE="https://www.amd.com/en/support/kb/release-notes/rn-amdgpu-unified-linux-21-30"
+SRC_URI="https://drivers.amd.com/drivers/linux/amdgpu-pro-${MY_PV}-ubuntu-20.04.tar.xz"
+S="${WORKDIR}"
+
+RESTRICT="bindist mirror fetch strip"
+
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+QA_PREBUILT="
+	/usr/lib64/amdvlkpro64.so*
+	/usr/lib/amdvlkpro32.so*
+"
+
+pkg_nofetch() {
+	einfo "Please download:"
+	einfo "    Radeon™ Software for Linux® version ${MY_PV_MAJOR}.${MY_PV_MINOR} for Ubuntu 20.04.3"
+	einfo "from ${HOMEPAGE} and place them in your DISTDIR directory."
+}
+
+src_unpack() {
+	default
+
+	if usex "abi_x86_64" ; then
```

Comment:
I wouldn't bother quoting "abi_x86.."

Reply:
I do it to get the pretty syntax highlighting in my editor, I will remove it if you insist x)